### PR TITLE
Bug/64 cannot find recipient

### PR DESCRIPTION
**Pull request by theblockstalk**

### CODE REVIEW

**Review by sadiabbasi — APPROVED** [review]

File: src/accounts/molecules/ConnectionError.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            Please open {settings.config.appName} on your phone and login as{" "}
+            <u>{username}</u>.
+          </TP>
+        </>
+      ) : (
+        <>
+          <TP className="margin-top">
+            Could not connect to the {settings.config.appName}.
+          </TP>
+          <TP>Please try again.</TP>
```
Comment (sadiabbasi) [inline src/accounts/molecules/ConnectionError.tsx:26]: In connection error shouldn't we redirect user back to login?

Comment (theblockstalk) [reply src/accounts/molecules/ConnectionError.tsx:26]: how about we give a button/link that lets them go back?
Automatically might be confusing.

Comment (sadiabbasi) [reply src/accounts/molecules/ConnectionError.tsx:26]: yess that is the same thing i was thinking about or we can make "Please try again" clickable

Comment (theblockstalk) [reply src/accounts/molecules/ConnectionError.tsx]: done

**Pull request merged**